constraint_utils/constraint_basics.py

```diff
@@ -722,6 +777,16 @@ def build_plateau_models_pop(ht: hl.Table, weighted: bool = False) -> Dict[str,
 
 
 def get_all_pop_lengths(ht, prefix: str = 'observed_', pops: List[str] = POPS, skip_assertion: bool = False):
+    """
+    Get the minimum array length for each annotation in `ht` that is defined by the combination of `prefix` 
+    and each population in `pops`.
```

Comment:
I know I wrote this, but I guess it should be only one line for the description so maybe something like this:
```suggestion
    Get the minimum array length for specific per population annotations in `ht`.
    
    The annotations are specified by the combination of `prefix` and each population in `pops`.
```